helpers/env_vars.go

```diff
@@ -53,27 +60,88 @@ var (
 	TICSecretEnvVar = "TIC_SECRET"
 )
 
-// EnvVars holds all the environment variable values that a non-test server should have.
+// EnvVars provides a convenient method to access environment variables
 type EnvVars struct {
-	ClientID        string
-	ClientSecret    string
-	Hostname        string
-	LoginURL        string
-	UAAURL          string
-	APIURL          string
-	LogURL          string
-	PProfEnabled    string
-	BuildInfo       string
-	NewRelicLicense string
-	SecureCookies   string
-	LocalCF         string
-	SessionBackend  string
-	SessionKey      string
-	BasePath        string
-	SMTPHost        string
-	SMTPPort        string
-	SMTPUser        string
-	SMTPPass        string
-	SMTPFrom        string
-	TICSecret       string
+	path []EnvLookup
+}
+
+// Get returns value for key if present, else returns defaultVal if not found
+func (el *EnvVars) Get(key, defaultVal string) string {
+	rv, found := el.load(key)
+	if !found {
+		return defaultVal
+	}
+	return rv
+}
+
+// MustGet will panic if value is not set, otherwise it returns the value.
+func (el *EnvVars) MustGet(key string) string {
+	rv, found := el.load(key)
+	if !found {
+		panic(&ErrMissingEnvVar{EnvVar: key})
+	}
+	return rv
+}
+
+// BoolGet returns true if and only the value "true" or "1" is set for the key, else defaults to false
+func (el *EnvVars) BoolGet(key string) bool {
+	val, _ := el.load(key)
+	return val == "true" || val == "1"
```

Comment:
@aeijdenberg if you wanted to be a little more flexible here you could use `strconv.ParseBool`: https://golang.org/src/strconv/atob.go?s=351:391#L1

Reply:
Fixed as suggested - noting this is a slight change from previous behaviour - it now accepts a few additional values for `true`, and will now panic if set but not parsable as `true` or `false`.